### ISSUE TITLE
Issues with operands on the stack in lambdas

### ISSUE THREAD

**Issue by hawkw** [hawkw/seax#75]
Both SVM lambda integration targets now fail with the following messages:

```
failures:

---- run_lambda_ap stdout ----
    thread 'run_lambda_ap' panicked at '[fatal][ADD]: expected first operand, found None
', seax_svm/src/lib.rs:218


---- run_nested_lambda stdout ----
    thread 'run_nested_lambda' panicked at '[fatal][SUB]: expected second operand, found None
', seax_svm/src/lib.rs:231



failures:
    run_lambda_ap
    run_nested_lambda
```

I'm not entirely sure what's up here, but I have the strong sense that it has to do with the way lambda parameters are assembled into a closure.

**Comment by hawkw** [comment]
This issue was fixed in 18b5459423004415efdf4f57c7e993f3697b7ae3

**Comment by hawkw** [comment]
With that said, there is still one more weird closure building issue (closure construction is, evidentially, a Hard Problem In SVM), but it only breaks the nested lambdas target (for reasons currently unknown to me).
